### PR TITLE
docs: Access Log

### PR DESCRIPTION
**Pull request by gavindsouza**
Documented Access Log added in the [Frappe Framework](https://github.com/frappe/frappe/pull/8014)

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/using-erpnext/index.md [hunk 1]
```diff
@@ -15,28 +15,29 @@ effectively manage your data and collaborate with your co-workers.
 
 ## Topics
 1. [To Do](/docs/user/manual/en/using-erpnext/to-do)
-1. [Restore Deleted Docs](/docs/user/manual/en/using-erpnext/restore-deleted-docs)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/index.md]: undo this, makes no difference and was set this way for easy re-arranging

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<img class="screenshot" alt="Acess Log: General Ledger Report ERPNext" src="{{docs_base_url}}/assets/img/access-log/access-log-003.png">
+
+Buttons are enabled dynamically for easy access with relevant data, keeping simplicity at the core.
+
+<img class="screenshot" alt="Acess Log: Sales Invoice Frappe Custom Application" src="{{docs_base_url}}/assets/img/access-log/access-log-001.png">
+
+Events such as [raw printing](/docs/user/manual/en/setting-up/print/raw-printing) save information along with the chosen template for the document.
+
+<img class="screenshot" alt="Acess Log: HTML to PDF Printing" src="{{docs_base_url}}/assets/img/access-log/access-log-004.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: image top needs cropping

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+show_sidebar: 0
+
+metatags:
+ description: Access Log is a security feature that logs all data exports in the form of printing of Forms and reports, private file downloading and exporting reports in excel/csv formats.
+ keywords: frappe, access log, erpnext v12, erpnext release notes, erpnext new features, erp, open source erp, free erp, security
+---
+
+# Access Log
+
+Access Log is a security feature that logs all data exports by all users in the system. This feature is enabled by default from **v13**.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]:
```suggestion
**Access Log is a security feature that logs all data exports by all users in the system.**
> This feature is enabled by default from v13.
```

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+> Home > Users and Permissions > Logs > Access Log
+
+Access logs are created in the following events:
+
+ - Printing of all Forms and Reports
+ - Private file downloading
+ - Exporting via XLSX/CSV formats
+
+
+<img class="screenshot" alt="Acess Log: ERPNext List View" src="{{docs_base_url}}/assets/img/access-log/access-log-002.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: add a line to introduce the image

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+The general List view provided out of the box by [Frappe](https://frappe.io/frappe).
+
+<img class="screenshot" alt="Acess Log: General Ledger Report ERPNext" src="{{docs_base_url}}/assets/img/access-log/access-log-003.png">
+
+Buttons are enabled dynamically for easy access with relevant data, keeping simplicity at the core.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: unclear, what do you mean?

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+<img class="screenshot" alt="Acess Log: Sales Invoice Frappe Custom Application" src="{{docs_base_url}}/assets/img/access-log/access-log-001.png">
+
+Events such as [raw printing](/docs/user/manual/en/setting-up/print/raw-printing) save information along with the chosen template for the document.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]:
```suggestion
Events such as [Raw Printing](/docs/user/manual/en/setting-up/print/raw-printing) save information along with the chosen template for the document.
```

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md (hunk 5 above)
Comment (gavindsouza) [reply erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: I've tried explaining it further

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 7]
```diff
@@ -10,8 +10,9 @@ metatags:
 
 # Access Log
 
+> Introduced in Version 13
```
Comment (gavindsouza) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: rearranged with reference to [this](https://erpnext.com/docs/user/manual/en/setting-up/automation/milestone-tracker)

File: erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+<img class="screenshot" alt="Acess Log: General Ledger Report ERPNext" src="{{docs_base_url}}/assets/img/collaboration-tools/access-log-003.png">
+
+Similarly, a **Show Document** button is generated in case of events related to documents directly such as document Printing and Private File download.
+
+<img class="screenshot" alt="Acess Log: Sales Invoice Frappe Custom Application" src="{{docs_base_url}}/assets/img/collaboration-tools/access-log-001.png">
+
+Events such as [Raw Printing](/docs/user/manual/en/setting-up/print/raw-printing) save information along with the chosen template for the document.
+
+<img class="screenshot" alt="Acess Log: HTML to PDF Printing" src="{{docs_base_url}}/assets/img/collaboration-tools/access-log-004.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md]: inconsistent padding width wise

**Pull request merged**